### PR TITLE
Rank revamp

### PR DESCRIPTION
**Pull request by vaelliott**
I think i've completed the changes required for the rank tab although I see there are merge conflicts with development again, would you expected or do I need to look into why?

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -1114,7 +1114,8 @@ function(input, output, session) {
     showModal(modalDialog(
       title = "How to use this chart",
       p("The trend chart is designed to explore how a single indicator has changed over time for one or more geograpical area."),
-      p(column(7,img(src="help_trend_chart2.png")),
+      p(column(7,
+               tags$a(img(src="help_trend_chart2.png"))),
```
Comment (jvillacampa) [inline server.R]: Is this tags$a needed?

File: ui.R [hunk 2]
```diff
@@ -349,62 +349,58 @@ tabPanel("Trend", icon = icon("area-chart"), value = "trend",
...
+                  div(title="Use this option to change the type of geography displayed in the chart. Some indicators are not be available for all geography types.  See technical information to find out which geographies indicators are available for. ",
+                  uiOutput("geotype_ui_rank")),
+                  div(title="There are too many hscp localities or IZs to show in the rank chart a selection must be made to limit localities or IZs to only those within a parent area",
                   conditionalPanel( #Conditional panel for extra dropdown for localities & IZ
                     condition = "input.geotype_rank == 'HSC locality' | input.geotype_rank == 'Intermediate zone' ",
-                    selectInput("loc_iz_rank", label = "Partnership for localities/intermediate zones",
-                                choices = partnership_name))
+                    selectInput("loc_iz_rank", "Step 2b. Select a region for localities or intermediate zones",
```
Comment (jvillacampa) [inline ui.R]: This is good, I will change it in the heatmap too

File: ui.R [hunk 3]
```diff
@@ -349,62 +349,58 @@ tabPanel("Trend", icon = icon("area-chart"), value = "trend",
...
-                  uiOutput("year_ui_rank"), 
-                  awesomeRadio("comp_rank", label = "Compare against:",
-                               choices = list("Area" = 1, "Time" = 2), 
-                               selected = 1, inline=TRUE, checkbox=TRUE),
+                  div(title="Use this option to change the time period presented in the chart and map",
+                      uiOutput("year_ui_rank"))), 
+           column(width = 3,
+                  div(title="This option will change whether the chart compares areas to another area (e.g. the Scotland average) or against a different time period (e.g. figures for the year 2017 compared to the year 2010).",
+                      awesomeRadio("comp_rank", label =shiny::HTML("<p>Step 4. Select to compare by:<br/><br/></p>"), #br required to try and keep alignment across columns
+                                   choices = list("Area or"= 1, "Time" = 2),
```
Comment (jvillacampa) [inline ui.R]: I will change the heatmap text so it is the same as here

File: server.R [hunk 4]
```diff
@@ -1478,7 +1531,14 @@ function(input, output, session) {
   output$rank_title <- renderText( paste0(input$indic_rank) )
   
   output$rank_subtitle <- renderText({ make_rank_subtitle()  })
-  
+ 
+  #visible summary of ui main panel to guide users
+  make_rank_summary <- function() {
+    case_when(input$comp_rank == 1 ~ paste0("The bar chart and map below both show how areas of the same type (e.g. NHS board) compare to each other for a particular indicator."),
+              input$comp_rank == 2 ~ paste0("The chart below is called a lollipop chart, it shows how areas compare with each other and also how each area has changed over time. The map shows a static comparison of areas for the latest time period."))
```
Comment (jvillacampa) [inline server.R]: This is wrong. When the time comparison selected the map shows a comparison against the baseline year chosen.

Comment (vaelliott) [reply server.R]: good point, i've fixed this

**Comment by vaelliott** [comment]
Chose not to fix the comparator issues still present (e.g. Drug mortality if you chose a CA comparator) at this stage. Better to attempt adjusting the geographies available for these indicators during update process first

**Pull request merged**